pkg/virt-controller/services/template_test.go

```diff
@@ -61,6 +61,8 @@ import (
 	"kubevirt.io/kubevirt/tools/vms-generator/utils"
 )
 
+var testHookSidecar = hooks.HookSidecar{Image: "test-image", ImagePullPolicy: "test-policy"}
```

Comment:
nit: Please move closer to usage (near testSidecarCreator())

Reply:
Not possible, it is used inside the test containers and on a root function.

Reply:
But isnt it global, outside any tests container?